@here/harp-map-controls/lib/CameraKeyTrackAnimation.ts

```diff
@@ -0,0 +1,289 @@
+/*
+ * Copyright (C) 2020 HERE Europe B.V.
+ * Licensed under Apache 2.0, see full license in LICENSE
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { GeoCoordinates, GeoCoordLike } from "@here/harp-geoutils";
+import { LookAtParams, MapView, MapViewEventNames, MapViewUtils } from "@here/harp-mapview";
+import { LoggerManager } from "@here/harp-utils";
+import THREE = require("three");
+
+const logger = LoggerManager.instance.create("CameraKeyTrackAnimation");
+
+export interface ControlPointOptions extends Partial<LookAtParams> {
+    /**
+     * @override
+     */
+    target: GeoCoordinates | GeoCoordLike;
+    /**
+     * A timestamp for the [[ControlPoint]], describes when during the animation the
+     * [[ControlPoint]] should be reached, in seconds
+     */
+    timestamp: number;
+    /**
+     * @override
+     */
+    tilt?: number;
```

Comment:
Why are these ones here again? They are already part of LookAtParams.

Reply:
Yes, but LookAtParams is an interface it still needs implementation.... (typescript will complain otherwise)

Reply:
but `ControlPointOptions` is also just an interface

Reply:
so, found a solution, the types did not all match, but do now, and I used Pick over Partial, as it only adds the parameters needed.